Fix imports of own packages

geigi opened the pull request:
The module name should not include the full GitHub path because local development will not be possible.
Instead of using the local files go will always use the sources from GitHub.
Removing the full GitHub path from the module name solves this issue.

geigi commented:
This fixes issues introduced by https://github.com/FactoryCampus/radau/pull/4

mraerino commented:
i think it's pretty common to add the full path to the module name.

this should not impact local development, since with go modules you're expected to develop outside your GOPATH and the go toolchain will take care of the imports, no matter what the module name really is.

generally, keeping the full repo path in the module name makes it easier to port bits of code to another project without needing to change the import paths. it also avoid name clashes between packages.

geigi committed:
Fix own package imports
The module name differed from the imports used which caused go to always import the own package from GitHub and ignoring the local one.

geigi commented:
Thanks for the explanation! 
I found the issue which was causing this behavior. The module name is `github.com/FactoryCampus/radau` but the imports used all lowercase `factorycampus`. radau was then added as a dependency of itself, but using the all lowercase path.
I've force pushed a new commit to this PR. I've renamed the `factorycampus` imports to  `FactoryCampus`. I wasn't sure weather to use `FactoryCampus` or `factorycampus` tough.

mraerino reviewed (approved):
that makes sense.
usually an artifact of macos having a case-insesitive filesystem.

i usually go with lowercase only, but sadly we can't really change the name of the org in github
